hatch/src/cli/commands/new.rs

```diff
@@ -80,10 +94,12 @@ impl<'command> Command<'command> for New {
       Ok(_) => {
         let version = self.project_version(args);
         let kind = self.project_kind(args);
+        let includes =  String::from("  - ") + &self.get_includes(args).join("\n  - ");
```

Comment:
```let includes =  String::from("  - ") + &self.get_includes(args).join("\n  - ");```
Please remove the extra space
```let includes = String::from("  - ") + &self.get_includes(args).join("\n  - ");```

Reply:
I think the spec puts two spaces. www.yaml.org/spec/1.2/spec.html. Scroll down to the Collections section.

Reply:
no theres a second space after the `let includes = `

Reply:
oh wow didn't even see that one thanks